### PR TITLE
Add last updated to CMS Dashboard

### PR DESCRIPTION
**Pull request by haworku**
## Summary
- remove CMS_DASHBOARD feature flag
- Add last updated column and in general move from looking at current revision to previous submitted revision for form data. That's something I overlooked in the first pass at the dashboard, likely this would cause bugs.

#### Related issues
https://qmacbis.atlassian.net/browse/MR-2466

#### Test cases covered
**Jest**
see `CMSDashboard.test.tsx` (web tests)
 - displays no submission text when no submitted packages exist
- displays submissions table when submitted packages exist
- displays submissions table with expected headers
- displays submissions table excluding any in progress drafts
- displays submission type as expected for current revision that is submitted/resubmitted
- displays the expected program tags for current revision that is submitted/resubmitted
- displays each health plan package status tag as expected
- displays name, type, programs and last update based on previously submitted revision for UNLOCKED package

**Cypress**
- CMS user can view submission in the CMS dashboard

## QA guidance
- Please test this work thoroughly with unlock and resubmit to make sure that state draft edits  data is not appearing in the dashboard. This is most relevant when an unlocked package is in progress of being edited
- check that last updated column displays as expected

### CODE REVIEW

File: services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx [hunk 1]
```diff
@@ -110,39 +101,64 @@ export const CMSDashboard = (): React.ReactElement => {
...
+                    !previousRevision?.node?.unlockInfo?.updatedAt &&
+                    !previousRevision?.node?.submitInfo?.updatedAt
+                ) {
+                    recordJSException(
+                        `indexHealthPlanPackagesQuery: Error finding updatedAt for an UNLOCKED SUBMISSION. Check the revision data is properly formatted. ID: ${sub.id}`
+                    )
+
+                    return
+                }
+                lastUpdated =
```
Comment (haworku) [inline services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx:137]: TODO: add a comment here.Last updated at value should be CMS unlock time (if its unlocked) or else state submit time (if its just submitted), fall back to last updated at  for the current revision (likely we wouldn't need that but just to have something there).

**Review by JasonLin0991 — CHANGES_REQUESTED** [review]
Seems to not move the recently resubmitted submission up to the top of the list.

https://user-images.githubusercontent.com/98117700/184021478-366f1661-d45c-466c-bd7c-36c9f749f7fa.mov

**Comment by haworku** [comment]
Thanks @JasonLin0991 must have broken something. Will look at this and also if there's a way to catch it through tests

File: services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx [hunk 2]
```diff
@@ -103,11 +103,12 @@ export const CMSDashboard = (): React.ReactElement => {
...
 
             // Calculate proper submission data to display in each row
-            // if UNLOCKED package, form data could be in progress of being edited, show the static form data from the last submitted revision
-            // also, use lastUpdated from unlockData (if unlocked) or last submit (if initial submission)
-            // otherwise, use data from current revision for Submitted or Resubmitted packages
+            // if Unlocked package, form data could be in progress of being edited, show the static form data from the last submitted revision
+            // otherwise, use current data for Submitted or Resubmitted packages
             let packageDataToDisplay = currentFormData
-            let lastUpdated = currentFormData.updatedAt
+            let lastUpdated =
```
Comment (haworku) [inline services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx:109]: Consider making a utility function to compare a set of dates and pull most recent as last updated

**Review by JasonLin0991 — APPROVED** [review]
Works great and thanks for those additional changes!

**Pull request merged**